elbs/tf/all_elbs.tf

```diff
@@ -93,6 +96,7 @@ module "careers" {
 }
```

Comment:
Can you please add
` health_check_http_path = "/healthz/"` 

now that careers has a healthcheck endpoint?